internal/clients/team3/clientiigo.go

```diff
@@ -339,3 +339,10 @@ func copyRulesMap(inp map[string]rules.RuleMatrix) map[string]rules.RuleMatrix {
 	}
 	return newMap
 }
+
+func safeDivResources(numerator shared.Resources, denominator shared.Resources) shared.Resources {
+	if denominator != 0 {
+		return numerator / denominator
+	}
+	return numerator
```

Comment:
I'd log when this happens tbh -- btw I like returning 0 instead.

Reply:
It's our agent strat to do the illogical thing